charts/partials/templates/_proxy.tpl

```diff
@@ -39,8 +39,16 @@ env:
 - name: LINKERD2_PROXY_IDENTITY_DIR
   value: /var/run/linkerd/identity/end-entity
 - name: LINKERD2_PROXY_IDENTITY_TRUST_ANCHORS
+{{- if and (.Identity.Issuer) (eq .Identity.Issuer.Scheme "linkerd.io/cert-manager") }}
```

Comment:
FYI, we do not really support this scheme atm. The schemes we support are : 

- `linkerd.io/tls`
- `kubernetes.io/tls`